pubnub/managers.py

```diff
@@ -89,6 +93,26 @@ def _stop_heartbeat_timer(self):
             self._timer = None
 
 
+class LinearDelay:
```

Comment:
Is it possible to provide custom delay other than `2`?

Reply:
No, not really. Should we add this option?

Reply:
added in latest commit. thanks for pointing that out ;)